### DISCUSSION TITLE
Any way to make an item static in case other items are being moved? (without using static property)

### DISCUSSION THREAD

**Discussion by ghost** [jbaysolutions/vue-grid-layout#552] [Q&A] [answered]
Hi!

Is there any way to make an item static in case other items are being moved? (without using static property) . I want the items to be draggable but not change their slot if other items are deleted from the list.

Thanks!

**Comment by gmsa** [comment, marked as answer]
Disable [vertical-compact](https://jbaysolutions.github.io/vue-grid-layout/guide/properties.html#verticalcompact) and it should work.

**Reply by ghost** [reply]
k, thanks.
